src/components/Banner/Banner.style.ts

```diff
@@ -56,7 +56,11 @@ export const ButtonContainer = styled(Grid)`
   }
 `;
 
-export const SurveyButton = styled(MuiButton)`
+export const ContainedButton = styled(MuiButton).attrs(props => ({
```

Comment:
Not sure what "contained button" means.  Add a comment and/or rename?

Reply:
Is one of the variants defined by Material UI (https://material-ui.com/components/buttons/#contained-buttons), I renamed it because we were using `SurveyButton` in non-survey contexts

Reply:
🙏